seed/csharp-sdk/idempotency-headers/src/SeedIdempotencyHeaders/Core/IIdempotentRequestOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace SeedIdempotencyHeaders.Core;
+
+internal interface IIdempotentRequestOptions : IRequestOptions
+{
+    public string IdempotencyKey { get; init; }
+    public int IdempotencyExpiration { get; init; }
+    internal Headers GetIdempotencyHeaders();
+}
```

Comment:
what's the benefit of having the interface here as opposed to `IdempotentRequstOptions` just inheriting from `IRequestOptions` ?

Reply:
Another options variation may be introduced in the future that may inherit from `IRequestOptions` or from `IIdempotentRequestOptions`